chore: public api in readme (#281)
superfaceai/one-sdk-js: README.md

You can find more options for reaching us on the [Support page](https://superface.ai/support).

## Public API

> If you decide to use some function from this package in your own code you definetly can but be aware that not every part of package is version controlled.

Reviewer: ```suggestion
```

> Only function and API of entities below is under version control:

Reviewer: ```suggestion
Only functions and APIs of entities below are a part of the public API, and can be safely relied upon not to break between semver-compatible releases.

Using other parts of this package is at your own risk.
```

Reviewer: what do you think @Jakub-Vacek? :)

Reviewer: rationale - "you can use this package, but beware, there are dragons somewhere!" is not really the message IMHO, it's more that "if you want to use this package, there's A, B & C and everything else is dragons"

Author: I like it :) 
A, B & C are also dragons but we will try to keep them dragons😅

* SuperfaceClient API
* Profile API
* UseCase API

> * SuperJsonDocument object

Reviewer: ```suggestion
* SuperJsonDocument Object
```

You can find more options for reaching us on the [Support page](https://superface.ai/support).

## Public API

Only functions and APIs of entities below are a part of the public API, and can be safely relied upon not to break between semver-compatible releases.

Using other parts of this package is at your own risk.

* SuperfaceClient API
* Profile API
* UseCase API
* SuperJsonDocument Object
* Result API

> Use of public API is described in [reference](https://superface.ai/docs/reference/one-sdk).

Reviewer: ```suggestion
Use of public APIs is described in the [reference](https://superface.ai/docs/reference/one-sdk).
```
